### ISSUE TITLE
_gid cookie not linked to Google Analytics

### ISSUE THREAD

**Issue by epigeyre** [AmauriC/tarteaucitron.js#40]
Hello !

First of all, thanks for your plugin, it works great! 

I've just noticed however that the `_gid` cookie is not linked to Google Analytics (it's a new Universal Analytics cookie since April 2017 I think). This brings two issues:

- The `_gid` cookie owner is my domain name instead of Google Analytics Universal ;
- The `_gid` cookie doesn't get deleted when you disable GA Universal service.

Everything seems to be solved by adding the cookie name `_gid` into the array **line 799** of the `tarteaucitron.services.js` file.

**Comment by AmauriC** [comment]
Hello !

Can you submit a pull request? That's the solution 👍

**Comment by corethical** [comment]
[Sans titre.txt](https://github.com/AmauriC/tarteaucitron.js/files/1194118/Sans.titre.txt)

**Comment by corethical** [comment]
I'm sorry I'm only trying to understand how to code, but i'm right at the begining, only one week I'm studying Java...I'll start in september an online Webmaster certificate, I know that It won't be easy at all...But I'll try, it's 18 months after you must pas the exam to get certification.
So it's been a week I'm trying to understand something of Java coding. must say for the moment I'm just lost...But I think it's normal when you know that that 6 months ago, I did not know what coding was?
I've found this cookie, I couldn't remove from Chrome MAC osx sierra, so I import it, and I'd like to know if someone which a little time, to expain me, what it is? A script, I really don't know at all informations that it's got in.
thank you very for any reply. It'll help me a lot to start understanding something about code.

Best regards. Stephane from Switzerland
